compiler/rustc_mir_transform/src/check_unsafety.rs

```diff
@@ -106,7 +106,7 @@ impl<'tcx> Visitor<'tcx> for UnsafetyChecker<'_, 'tcx> {
             }
 
             // Move to above list once mir construction uses it.
```

Comment:
Please do what the comment says ^^

Reply:
I completely removed statement match as it no longer has any use

Reply:
Oh but it does ^^ it makes sure when we add new variants that we address them here